nautobot/project-static/css/dark.css

```diff
@@ -36,6 +36,7 @@ html[data-theme="dark"] .container-fluid.wrapper {
 
 /* Lighter than the background */
 html[data-theme="dark"] .navbar,                          /* Top menu navbar */
+html[data-theme="dark"] .footer,                          /* Bottom footer bar */
 html[data-theme="dark"] .footer .row {                    /* Bottom footer bar */
```

Comment:
Is it correct to have both of these?

Reply:
If we don't set the footer row explicitly it will inherit the background color property from this section in `base.css` instead of from the parent element:
```css
.footer .row {
    background-color: #f5f5f5;
}
```

The footer row overlaps the entire footer in every case except on the graphql ui when the viewport is wide enough so that the footer doesn't overflow. In that case if the footer color isn't set you'd see a 1 or 2 pixel white line at the very bottom of the page. I'm not sure why the graphql ui is different.

Reply:
Actually the base.css change was something I changed in #2747. Let me see if I can condense this.

Reply:
The base.css change was the fix for #2552. So both are needed and I just updated the light mode theme to fix the tiny white line in the graphql that was so subtle I didn't notice it until I zoomed it to 800%